src/main/groovy/jenkins/automation/rest/RestApiScriptRunner.groovy

```diff
@@ -28,5 +29,5 @@ params['JAC_HOST'] = System.getProperty('JAC_HOST') ?: 'aws'
 new FileNameFinder().getFileNames('.', pattern).each { String fileName ->
     println "\nprocessing file: $fileName"
     File file = new File(fileName)
-    DslScriptLoader.runDslEngine(file.text, jm)
+    new DslScriptLoader(jm).runScripts([new ScriptRequest(file.text)])
```

Comment:
Sorry I am new to jobs-dsl-core, noticed there is a better api to use:
`DslScriptLoader.runScript(string)`

I will update the code and push a new change